Reason: Fails to link with newer cmake
Source: Upstream
Upstream: Yes

From 3b54e05bfac5a4a0c03d40317959d5ccfd6d346c Mon Sep 17 00:00:00 2001
From: Christophe Giboudeaux <cgiboudeaux@gmx.com>
Date: Sun, 4 Dec 2011 17:33:10 +0100
Subject: [PATCH] Don't try to find BoostConfig.cmake. The variables names are
 not compatible with CMake's ones.

---
 CMakeLists.txt |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 8ab8930..bb466f4 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -171,6 +171,10 @@ set(XSLTPROC_EXECUTABLE "${HOST_BINDIR}/xsltproc.exe")
 endif(WINCE)
 
 #### Boost ####
+# In CMake >= 2.8.6, FindBoost.cmake tries to find BoostConfig.cmake which is
+# not compatible with CMake's FindBoost. Disable this function.
+set (Boost_NO_BOOST_CMAKE TRUE)
+
 if(MSVC)
   set (_ENABLE_EXCEPTIONS -EHsc)
 endif(MSVC)
-- 
1.7.7.4

